stock_traceability_operation/views/stock_production_lot_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- © 2015 Numérigraphe
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+
+<openerp>
+<data>
+
+<record model="ir.ui.view" id="stock_production_lot_traceability_operation">
+    <field name="name">Add operation traceability on lots</field>
+    <field name="model">stock.production.lot</field>
+    <field name="inherit_id" ref="stock.view_production_lot_form"/>
+    <field name="arch" type="xml">
+        <xpath expr="//button[@name='action_traceability']" position="attributes">
+            <attribute name="attrs">{'invisible': True}</attribute>
+        </xpath>
+        <xpath expr="//button[@name='action_traceability']" position="after">
+            <button name="action_traceability_operation" icon="fa-arrows-v" class="oe_stat_button"
+                string="Traceability" type="object" attrs="{'invisible': [('quant_ids','=',[])]}"/>
```

Comment:
Call it "Traceability (by operation)" or similar to differentiate it from the other button.